.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
```

Comment:
According to the instructions the .gitignore was supposed to be created and have all .env files excluded in the main branch, but not in the PR branch.